docs/en/contribution.md

```diff
@@ -1,7 +1,8 @@
 ---
 layout: default
 title: Contribution Guide
-permalink: /contribution/
+pagename: contribution
+redirect_from: "/contribution/"
```

Comment:
i don't think we need any redirection at this point. We haven't been sharing out any links yet. Please confirm with @tannazie .

Reply:
For now I removed the redirects, and also removed the plugin from `tk-doc-generator`.  I can put both back (or just the plugin) if we decide we want to do that easily by reverting the commits.  @tannazie do you think we can do without the redirects?

Reply:
We have shared some links to guides in tickets, but it's minimal, and the site is so small, that I think people can find what they're looking for (or ask us). Looking into the future, the time before this change will be very small in comparison to the time after it (hopefully!). As far as I'm concerned, we're good to move forward without redirects.

Reply:
Sounds Great! Thanks @tannazie !